src/views/home/index.tsx

```diff
@@ -42,6 +75,7 @@ export const HomeView = () => {
           <TokenIcon mintAddress={SRM_ADDRESS} /> SRM: {SRM?.balance} (
           {formatUSD.format(SRM?.balanceInUSD)})
         </h2>
+        <div>{userInfo && JSON.stringify(userInfo)}</div>

```

Comment:
i need to see how this looks like :D

Reply:
Just a string of the response! haha. Nothing big. This PR isn't intended to finish #10. I just want to update us with something we can possibly split the branch/issue on.

Reply:
yea seems like we could redesign on later tickets